satellite/payments/stripecoinpayments/service.go

```diff
@@ -837,10 +864,22 @@ func (service *Service) CreateInvoices(ctx context.Context) (err error) {
 func (service *Service) createInvoice(ctx context.Context, cusID string) (err error) {
 	defer mon.Task()(&ctx)(&err)
 
+	// get the first invoice item's period
+	iter := service.stripeClient.InvoiceItems.List(&stripe.InvoiceItemListParams{Customer: stripe.String(cusID)})
+	if !iter.Next() {
+		return Error.New("no invoice items found, no invoice to be created")
```

Comment:
If you won't to throw this error you might also want to remove error handling on line 887